### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by dujfalusi**

### CODE REVIEW

File: mvnw.cmd [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+if exist "%HOME%\mavenrc_post.bat" call "%HOME%\mavenrc_post.bat"
+if exist "%HOME%\mavenrc_post.cmd" call "%HOME%\mavenrc_post.cmd"
+:skipRcPost
+
+@REM pause the script if MAVEN_BATCH_PAUSE is set to 'on'
+if "%MAVEN_BATCH_PAUSE%" == "on" pause
+
+if "%MAVEN_TERMINATE_CMD%" == "on" exit %ERROR_CODE%
+
+exit /B %ERROR_CODE%
```
Comment (stanislavgrujic) [inline mvnw.cmd]: Why do we need these two scripts?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.execom</groupId>
+  <artifactId>hawaii</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>hawaii</name>
+  <description>Demo project for Spring Boot</description>
```
Comment (stanislavgrujic) [inline pom.xml]: Please update description to match the project description.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    <!-- Project lombok -->
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <optional>true</optional>
+      <version>1.16.22</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
```
Comment (stanislavgrujic) [inline pom.xml]: Why do you need jaxb?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        <configuration>
+          <jvmArguments>--add-modules java.xml.bind</jvmArguments>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.7.0</version>
+        <configuration>
+          <release>10</release>
```
Comment (stanislavgrujic) [inline pom.xml]: Please use property `java.version`.

File: .gitignore [hunk 5]
```diff
@@ -1,23 +1,24 @@
...
-*.war
-*.nar
-*.ear
-*.zip
-*.tar.gz
-*.rar
+# DEPENDENCIES
+**/node_modules
+/.metadata/
+/.metadata/
```
Comment (stanislavgrujic) [inline .gitignore]: Is this line duplication intentional?

File: README.md [hunk 6]
```diff
@@ -1 +1,42 @@
...
+### Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+#### Prerequisites
+What things you need to setup before starting server
+
+- Install [Java10](https://docs.oracle.com/javase/10/install/overview-jdk-10-and-jre-10-installation.htm#JSJIG-GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A) and set the JAVA_HOME Environment Variable
+- Install [Maven](https://www.mkyong.com/maven/how-to-install-maven-in-ubuntu/)
+- Install [MySQL](https://www.digitalocean.com/community/tutorials/how-to-install-mysql-on-ubuntu-16-04)
+- Install [Tomcat9](https://tecadmin.net/install-tomcat-9-on-ubuntu/)
```
Comment (stanislavgrujic) [inline README.md]: Please create a section for Ubuntu, to make it clear, that the configuration is for ubuntu.

File: README.md [hunk 7]
```diff
@@ -1 +1,42 @@
...
+- Install Lombok plugin in IntelliJ
```
Comment (stanislavgrujic) [inline README.md]: This goes into common section.

File: src/main/java/eu/execom/hawaii/ServletInitializer.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
+
+	@Override
+	protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+		return application.sources(HawaiiApplication.class);
+	}
+
+}
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/ServletInitializer.java]: Please format the file properly.

File: src/test/java/eu/execom/hawaii/HawaiiApplicationTests.java [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class HawaiiApplicationTests {
+
+	@Test
+	public void contextLoads() {
+	}
+
+}
```
Comment (stanislavgrujic) [inline src/test/java/eu/execom/hawaii/HawaiiApplicationTests.java]: Please format the file properly.

File: mvnw.cmd (hunk 1 above)
Comment (dufajlusi) [reply mvnw.cmd]: That is maven startup script which is generated automatically with a project. Will be deleted.

Comment (stanislavgrujic) [reply mvnw.cmd]: Leave it, I just wanted to know what's its purpose...

File: pom.xml (hunk 3 above)
Comment (dufajlusi) [reply pom.xml]: JDK 9 has deprecated java.xml.bind module and has removed from default classpath, javax.xml.bind is sub package of Module java.xml.bind

So Module javax.xml.bind will not be available on classpath by default in JAVA 9.

http://www.baeldung.com/java-9-jaxbexception

File: .gitignore (hunk 5 above)
Comment (dufajlusi) [reply .gitignore]: No, will be removed for now until frontend guys define a new one(after adding fronted initial-setup).

**Pull request merged**